lib/app_profiler/profile.rb

```diff
@@ -41,6 +41,7 @@ def upload
       AppProfiler.storage.upload(self).tap do |upload|
         if upload && defined?(upload.url)
           AppProfiler.logger.info("[Profiler] uploaded profile profile_url=#{upload.url}")
+          AppProfiler.logger.info("[Profiler] Profile available at profile_url=#{AppProfiler.profile_url(upload)}")
```

Comment:
I think we're talking about 2 different things in this log message. The profile_url (the location of the profile on the storage service) and the speedscope_url (the address to actually view the profile). I think this would make more sense:

```
AppProfiler.logger.info("[Profiler] uploaded profile profile_url=#{upload.url} speedscope_url=#{AppProfiler.profile_url_formatter.call(upload)}")
```

It would be nice if we could define `#speedscope_url` on the upload object, but this would require a larger refactor of the storage services.